test/benchgc/src/benchgc/util.py

```diff
@@ -30,6 +30,7 @@
 ERROR_OUTPUT_VERBOSE = 4  # + print all error data points if failed
 OUTPUT_VERBOSE = 5  # + print all result including passed tensor
 INPUT_VERBOSE = 6  # + print input torch tensors
+PIPELINE_VERBOSE = 7  # + print ir when running pipeline
```

Comment:
Print IR verbose should be a new flag, do not add this feature in verbose.

Reply:
fixed